release-team/role-handbooks/ci-signal/README.md

```diff
@@ -13,7 +13,7 @@ CI Signal lead assumes the responsibility of the quality gate for the release. T
   - Understand and fix the test for the current release
   - Understand if the test needs to be release blocking
   - Work with SIG-Testing on any possible test infra improvement to help improve test pass rate
-- Making recommendations to SIG-Release for promoting and demoting release blocking and merge blocking tests as per the [Blocking tests Proposal](https://docs.google.com/document/d/1kCDdmlpTnHPQt5z8JzODdFCc3T2D4MKR53twsDZu20c/edit)
+- Making recommendations to SIG-Release for promoting and demoting release blocking and merge blocking tests as per the [Blocking tests Proposal](/release-blocking-jobs.md)
```

Comment:
please change label to "Blocking Jobs Criteria", since it's not really a proposal anymore.

Reply:
Fixed